pkg/scheduler/reconcile_on_edgeplacement.go

```diff
@@ -64,12 +68,16 @@ func (c *controller) reconcileOnEdgePlacement(ctx context.Context, key string) e
 			}
 			_, err = c.edgeClusterClient.Cluster(ws.Path()).EdgeV1alpha1().SinglePlacementSlices().Create(ctx, sps, metav1.CreateOptions{})
 			if err != nil {
-				logger.Error(err, "failed creating singlePlacementSlice", "workspace", ws.String(), "name", sps.Name)
+				if !errors.IsAlreadyExists(err) {
+					logger.Error(err, "failed creating singlePlacementSlice")
```

Comment:
This is not really a failure, this is evidence that concurrent activity got the desired job done.